oak-blob-cloud/src/test/java/org/apache/jackrabbit/oak/blob/cloud/s3/TestS3DSWithSSES3.java

```diff
@@ -71,12 +86,17 @@ public void testDataMigration() {
             props.setProperty(S3Constants.S3_RENAME_KEYS, "true");
             ds = createDataStore();
 
+            Assert.assertNotEquals(null, ds);
+            rec = ds.getRecord(rec.getIdentifier());
+            Assert.assertNotEquals(null, rec);
             rec = ds.getRecord(rec.getIdentifier());
             Assert.assertEquals(data.length, rec.getLength());
             assertRecord(data, rec);
 
             randomGen.nextBytes(data);
-            ds.addRecord(new ByteArrayInputStream(data));
+            rec = ds.addRecord(new ByteArrayInputStream(data));
+            DataRecord rec1 = ds.getRecord(rec.getIdentifier());
+            Assert.assertEquals(rec.getLength(), rec1.getLength());
```

Comment:
assert on content missing. Why not use `org.apache.jackrabbit.oak.plugins.blob.datastore.DataStoreUtils#randomStream` consistently everywhere for generating random data stream